nucypher/characters/lawful.py

```diff
@@ -871,15 +873,17 @@ def retrieve():
 
         return controller
 
-
 class Ursula(Teacher, Character, Worker):
+
     banner = URSULA_BANNER
     _alice_class = Alice
 
     # TODO: Maybe this wants to be a registry, so that, for example,  NRN
     # TLSHostingPower still can enjoy default status, but on a different class  NRN
     _default_crypto_powerups = [SigningPower, DecryptingPower]
 
+    _pruning_interval = 20
```

Comment:
Any context for the decision for the default of `20 seconds`? If just pruning does it make sense to do it a little less often...per minute(?) - I mean who knows 🤔 😊 ... just asking the question. Presumably pruning is a fairly light operation anyway.

Reply:
Its a fairly arbitrary guess trying to consider the duration of a very short policy, and the probability of the kfrag being pruned in a cohort of ursulas for one policy.  

Your sense is that this is a bit too slow (I think I agree)?

Reply:
Not based on anything but a 60-second (per minute) interval seems fine, which would be less iterations of the task, but still effective enough - saves the task from having to run too often. But I defer to you on this one, you would know better than I would.

Reply:
Since it's just a very simple query, it seems reasonable to do it every 20-30 seconds, imho.

Reply:
:heavy_check_mark: